sys/riscv/riscv/pmap.c

```diff
@@ -4292,6 +4337,12 @@ pmap_mincore(pmap_t pmap, vm_offset_t addr, vm_paddr_t *pap)
 			val |= MINCORE_MODIFIED | MINCORE_MODIFIED_OTHER;
 		if ((tpte & PTE_A) != 0)
 			val |= MINCORE_REFERENCED | MINCORE_REFERENCED_OTHER;
+
+#if __has_feature(capabilities)
+		if ((tpte & PTE_CW) != 0)
+			val |= MINCORE_CAPSTORE;
```

Comment:
1. Why do we need a CAPSTORE when there are no LOAD/STORE/EXECUTE equivalents?
2. Why do we _not_ have MINCORE_MODIFIED_CAP(_OTHER)?

Reply:
I think the _OTHER names would be more accurate to what they mean if they were _ANY instead of _OTHER FWIW.  The descriptions in the manpage do match those semantics.  Given we only have 1 bit, I'm not quite sure what makes the most sense to export.  For debugging / testing purposes we may want to export one thing now, but the better question is what will any real-world software really want (e.g. what would an allocator want to aid any user-space portion of a revoker).